src/misc/simulation/fscommon/fsdirectories.cpp

```diff
@@ -263,6 +336,35 @@ namespace swift::misc::simulation::fscommon
         return exclude;
     }
 
+    const QStringList &CFsDirectories::msfs2024SimObjectsExcludeDirectoryPatterns()
+    {
+        static const QStringList exclude {
+            /*
+            "OneStore/asobo-discovery",
+            "OneStore/asobo-flight",
+            "OneStore/asobo-landingchallenge",
+            "OneStore/asobo-mission",
+            "OneStore/asobo-tutorials",
+            "OneStore/asobo-vcockpits",
+            "OneStore/asobo-simobjects",
+            "OneStore/asobo-services",
+            "OneStore/asobo-vcockpits",
+            "OneStore/asobo-l",
+            "OneStore/asobo-m",
+            "OneStore/asobo-vfx",
+            "OneStore/fs",
+            "OneStore/esd",
+            "OneStore/microsoft-airport",
+            "OneStore/microsoft-bushtrip",
+            "OneStore/microsoft-discovery",
+            */
```

Comment:
Remove if they are no longer required.

Reply:
will probably be valid again when Asobo activates the marketplace. Deleted for now. Template is available